simplq/src/components/pages/Home/LandingPage.jsx

```diff
@@ -6,7 +6,7 @@ import styles from './Home.module.scss';
 import MyQueues from './MyQueues';
 
 export default () => {
-  let subtitle = 'A long overdue alternative to physical queues';
+  let subtitle = 'Give Your Customers a Delightful Customer Waiting Experience.';
   const { user, isAuthenticated } = useAuth0();
```

Comment:
Let's remove the second cutomer? - Give Your Customers a Delightful Waiting Experience